src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -701,6 +705,60 @@ base::Expression opBuilderHelperAppendString(const std::any& definition)
         });
 }
 
+// field: +s_to_array/$field/[,| | ...]
```

Comment:
fix name or remove comment of helper, should it be `+s_append_split`, right?

Reply:
Helper name is `+s_to_array`, I could change the function name.